readme.md

```diff
@@ -64,12 +64,82 @@ money out of it yourself.
 
 To update your own instance, follow the instructions below.
 
+## Running with Docker
+
+You can use [Docker](https://www.docker.com) and
+[docker-compose](https://docs.docker.com/compose/) to pull or build
+and run a Monica image, complete with a self-contained MySQL database.
+This has the nice properties that you don't have to install lots of
+software directly onto your system, and you can be up and running
+quickly with a known working environment.
+
+Before you start, run `cp .env.example .env` and open `.env` in an
```

Comment:
I feel there is a step missing.

Do we need to git clone the folder? Do we need to run a special docker CLI instruction before?